extensions/wikia/ArticleComments/classes/ArticleCommentsAjax.class.php

```diff
@@ -36,6 +36,13 @@ static public function axSave() {
 
 		$errorResult = [ 'error' => 1 ];
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "error" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)